New Quickstart for golem-js@3 (#278)
golemfactory/golem-docs: src/markdoc/partials/quickstarts/js-quickstart.md

mkdir try_golem
cd try_golem
npm init
npm install @golem-sdk/golem-js

> npm install @golem-sdk/pino-logger

Author: I wonder if it would not be better to use our cli here, e.g. `npx golem-sdk new` .. ?

Reviewer: we would need to change it in all examples

```

{% alert level="warning" %}
Please note: This application requires Node.js version 18.0.0 or higher.

{% /alert %}

Create a file named `requestor.mjs` and copy the following content into it. The code engages 5 providers (up to 3 at the same time) to run a command on each of them. The command checks the CPU of the provider and prints it together with the node's name to your terminal.

> {% codefromgithub url="https://raw.githubusercontent.com/golemfactory/golem-js/mgordel/JST-926/new-quickstart/examples/docs-examples/quickstarts/quickstart/requestor.mjs" language="javascript" /%}

Author: we need to update the correct url

The output of the script should look very similar to the one below:

<!-- replace this png -->

> ![Output logs](/run_log.png)

Author: ..and update screenshot

{% docnavigation title="Next steps" %}

- [Quickstart Explained](/docs/creators/javascript/tutorials/quickstart-explained)

> <!-- rebuild this article -->

Reviewer: ```suggestion
```

{% /docnavigation %}

{% docnavigation title="See also" %}

> - [Overview of JS SDK components](/docs/creators/javascript) <!-- rebuild this article -->

Reviewer: ```suggestion
- [Overview of JS SDK components](/docs/creators/javascript) 
```

> - [Simple introduction to the Golem Network and JS SDK](/docs/creators/javascript/guides/golem-js) <!-- add content to this article-->

Reviewer: ```suggestion
- [Simple introduction to the Golem Network and JS SDK](/docs/creators/javascript/guides/golem-js)
```

{% docnavigation title="See also" %}

- [Overview of JS SDK components](/docs/creators/javascript) 

> - [Simple introduction to the Golem Network and JS SDK](/docs/creators/javascript/guides/golem-js)

Reviewer: ```suggestion
- [Simple introduction to the Golem Network and Resource Rental Model](/docs/creators/javascript/guides/golem-js)
```

> - [JS Task Model](/docs/creators/javascript/task-executor/task-model)

Reviewer: ```suggestion
- Introduction to [JS Task Model](/docs/creators/javascript/task-executor/task-model) suitable for for use cases involving the batch-map-reduce type of operations that can be distributed across many rented resources.
```
